misc/python/materialize/buildkite.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""Buildkite utilities."""
+
+import os
+
+from materialize import git
+
+
+def is_in_buildkite() -> bool:
+    return os.getenv("BUILDKITE", "false") == "true"
+
+
+def is_in_pull_request() -> bool:
+    return os.getenv("BUILDKITE_PULL_REQUEST", "false") != "false"
+
+
+def get_pull_request_base_branch(fallback: str = "main"):
+    return os.getenv("BUILDKITE_PULL_REQUEST_BASE_BRANCH", fallback)
+
+
+def get_pipeline_default_branch(fallback: str = "main"):
+    return os.getenv("BUILDKITE_PIPELINE_DEFAULT_BRANCH", fallback)
+
+
+def determine_merge_base(remote="origin") -> str:
```

Comment:
`get_merge_base` ? All the other methods at the same level or called by it are `get_`

Reply:
✅ Renamed.